pilot/pkg/model/context.go

```diff
@@ -165,6 +187,9 @@ func GetNetworkView(node *Proxy) map[string]bool {
 		return map[string]bool{UnnamedNetwork: true}
 	}
 
+	// TODO: this is only used to select CDS ( DNS-based ) - not clear when user will have DNS names
+	// bound to a network or how a node would be part of multiple networks. It may not be needed.
+
```

Comment:
we use it in multicluster as well as for egress gateways. I think frank was adding an example. Do not remove please.

Reply:
More like please remove this comment

Reply:
ok

Reply:
Removed the comment - but I still think this may need to be re-designed and cleaned up. Which is fine since this is an alpha feature, and hasn't gone trough scalability testing yet.